base/inc/widgets/base-slider.class.php

```diff
@@ -271,7 +277,10 @@ function render_frame( $i, $frame ){
 			<?php
 			$this->render_frame_contents( $i, $frame );
 			if( !empty( $background['videos'] ) ) {
-				$this->video_code( $background['videos'], array('sow-' . $background['video-sizing'] . '-element') );
+
+				$classes = array( 'sow-' . $background['video-sizing'] . '-element' );
+				$classes[] = ! empty( $controls['background_video_mobile'] ) ? 'sow-mobile-video_enabled' : '';
```

Comment:
Wrather wrap this in an `if` block to avoid outputting an empty class, i.e.
```
if ( ! empty( $controls['background_video_mobile'] ) ) {
    $classes[] = 'sow-mobile-video_enabled';
}
```

Reply:
_Wrather wrap_... Wow, that's a special kind of typo. 😂